javabuilder-authorizer/jwt_helper.rb

```diff
@@ -91,23 +92,23 @@ def get_public_key(standardized_origin)
     # tmp = route_arn.split(':')
     # api_gateway_arn = tmp[5].split('/')
     # stage_name = api_gateway_arn[1]
-    stage_name = ""
-    if standardized_origin == "localhost-studio.code.org"
-      stage_name = "development"
-    elsif standardized_origin == "staging-studio.code.org"
-      stage_name = "staging"
-    elsif standardized_origin == "levelbuilder-studio.code.org"
-      stage_name = "levelbuilder"
-    elsif standardized_origin == "test-studio.code.org"
-      stage_name = "test"
-    elsif standardized_origin == "studio.code.org"
-      stage_name = "production"
-    elsif standardized_origin.start_with?("adhoc-") && standardized_origin.end_with?("-studio.cdn-code.org")
-      stage_name = "adhoc"
+    stage_name = ''
+    if standardized_origin == 'localhost-studio.code.org'
+      stage_name = 'development'
+    elsif standardized_origin == 'staging-studio.code.org'
+      stage_name = 'staging'
+    elsif standardized_origin == 'levelbuilder-studio.code.org'
+      stage_name = 'levelbuilder'
+    elsif standardized_origin == 'test-studio.code.org'
+      stage_name = 'test'
+    elsif standardized_origin == 'studio.code.org'
+      stage_name = 'production'
+    elsif standardized_origin.start_with?('adhoc-') && standardized_origin.end_with?('-studio.cdn-code.org')
+      stage_name = 'adhoc'
     end
     # End of temporary code
 
-    public_key = ENV["rsa_pub_#{stage_name}"]
+    public_key = ENV.fetch("rsa_pub_#{stage_name}", nil)
```

Comment:
Just curious, what rule is forcing us to use `fetch` here?

Reply:
It's this one:

https://docs.rubocop.org/rubocop/cops_style.html#stylefetchenvvar

It does bring up a broader question though, this rule is currently "pending" -- there's a setting (`NewCops`) of whether you want to automatically apply new cops included in each release. Might be better to disable this setting, which would only apply new cops on a major release:

https://docs.rubocop.org/rubocop/versioning.html#pending-cops

Reply:
hmm interesting. This rule does seem good, but it would be annoying to get new rules all the time. Maybe we disable the setting?

Reply:
Yeah agree, I think I might disable it and then reset of the autogenerated changes (including the env ones here) and reapply rubocop's autoformatting